<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#Login{background-image: url(:/bg/p1)};
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{width:213px}</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="readerTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>读者</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout" stretch="3,3,2">
          <property name="spacing">
           <number>7</number>
          </property>
          <item>
           <widget class="LabeledEdit" name="usernanmeInput" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>5</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>用户名</string>
            </property>
            <property name="toolTipDuration">
             <number>1000</number>
            </property>
            <property name="statusTip">
             <string>请输入用户名</string>
            </property>
            <property name="whatsThis">
             <string>不要使用非法字符</string>
            </property>
            <property name="accessibleName">
             <string>1234565</string>
            </property>
            <property name="accessibleDescription">
             <string>6546543513</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LabeledEdit" name="passInput" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>5</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="leftMargin">
             <number>7</number>
            </property>
            <property name="topMargin">
             <number>7</number>
            </property>
            <property name="rightMargin">
             <number>7</number>
            </property>
            <property name="bottomMargin">
             <number>7</number>
            </property>
            <item>
             <widget class="InteractiveButtonBase" name="readerRegister">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="statusTip">
               <string>1234</string>
              </property>
              <property name="text">
               <string>注册</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="InteractiveButtonBase" name="readerForgotPass">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="statusTip">
               <string>1234</string>
              </property>
              <property name="text">
               <string>忘记密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="InteractiveButtonBase" name="readerLogin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="statusTip">
               <string>1234</string>
              </property>
              <property name="text">
               <string>登录</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="adminTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>管理员</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,3,2">
          <item>
           <widget class="LabeledEdit" name="usernanmeInput_2" native="true">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>用户名</string>
            </property>
            <property name="toolTipDuration">
             <number>1000</number>
            </property>
            <property name="statusTip">
             <string>请输入用户名</string>
            </property>
            <property name="whatsThis">
             <string>不要使用非法字符</string>
            </property>
            <property name="accessibleName">
             <string>1234565</string>
            </property>
            <property name="accessibleDescription">
             <string>6546543513</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LabeledEdit" name="passInput_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="leftMargin">
             <number>7</number>
            </property>
            <property name="topMargin">
             <number>7</number>
            </property>
            <property name="rightMargin">
             <number>7</number>
            </property>
            <property name="bottomMargin">
             <number>7</number>
            </property>
            <item>
             <widget class="InteractiveButtonBase" name="adminForgotPass">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>60</height>
               </size>
              </property>
              <property name="statusTip">
               <string>1234</string>
              </property>
              <property name="text">
               <string>忘记密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="InteractiveButtonBase" name="adminLogin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>60</height>
               </size>
              </property>
              <property name="statusTip">
               <string>1234</string>
              </property>
              <property name="text">
               <string>登录</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveButtonBase</class>
   <extends>QPushButton</extends>
   <header location="global">interactive_buttons/interactivebuttonbase.h</header>
  </customwidget>
  <customwidget>
   <class>LabeledEdit</class>
   <extends>QWidget</extends>
   <header location="global">labeled_edit/labelededit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>readerLogin</tabstop>
  <tabstop>readerForgotPass</tabstop>
  <tabstop>readerRegister</tabstop>
  <tabstop>adminLogin</tabstop>
  <tabstop>adminForgotPass</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
